scripts/clean_project.sh

```diff
@@ -5,7 +5,7 @@ set -e -o pipefail
 echo "Cleaning XpringKit Project"
 
 rm -rf *.xcodeproj
-rm -rf XpringKit/generated
+rm -rf -r XpringKit/generated/
```

Comment:
`-r` is already expressed in `-rf`

Reply:
It was the file name that caused an error. I'll rebuild here in a bit and post the error here.

Reply:
and thanks for the fyi on `-r` / `-rf`